wdio.conf.js

```diff
@@ -24,8 +24,41 @@ const config = {
     enabled: !process.env.TRAVIS,
   },
 
+  maxInstances: 1,
   webpackPort,
   webpackConfig,
+
+  beforeHook() {
```

Comment:
We should get terra-toolkit released so we can pull in the helper for this.

Reply:
I can go back and create a PR to clean this up once we get that released if you are OK with that

Reply:
+1, sounds good to me.